test/test.redispatch.js

```diff
@@ -0,0 +1,89 @@
+import QUnit from 'qunit';
+import videojs from 'video.js';
+import redispatch from '../src/redispatch.js';
+
+QUnit.module('Redispatch', {
+
+  beforeEach(assert) {
+
+    // Player event buffer.
+    // Mocked player pushes events here when they are triggered.
+    // redispatch helper returns event buffer after each redispatch.
+    let eventBuffer = [];
+
+    // Mocked player
+    this.player = {
+      trigger(event) {
+        eventBuffer.push(event);
+      },
+
+      currentSrc() {
+        return 'my vid';
+      },
+
+      ads: {
+        state: 'content-set',
+
+        snapshot: {
+          ended: false,
+          currentSrc: 'my vid'
+        },
+
+        videoElementRecycled() {
```

Comment:
would be really, really nice to have another test for the `return true` case at the same time.